cmd/otelcontribcol/go.mod

```diff
@@ -438,7 +438,7 @@ require (
 	github.com/influxdata/go-syslog/v3 v3.0.1-0.20210608084020-ac565dc76ba6 // indirect
 	github.com/influxdata/influxdb-observability/common v0.5.2 // indirect
 	github.com/influxdata/influxdb-observability/influx2otel v0.5.2 // indirect
-	github.com/influxdata/influxdb-observability/otel2influx v0.5.2 // indirect
+	github.com/influxdata/influxdb-observability/otel2influx v0.0.0-20230705233630-3cfabd1d00b9 // indirect
```

Comment:
should this not be `v0.5.2-1...` or something like it?

Reply:
Fixed. (You forced me to finally learn how that works -- not complicated! https://go.dev/ref/mod#pseudo-versions )